apps/admin-x-activitypub/src/components/layout/Sidebar/Sidebar.tsx

```diff
@@ -2,38 +2,35 @@ import * as React from 'react';
 import Recommendations from './Recommendations';
 import SidebarButton from './SidebarButton';
 import {Button, LucideIcon, Separator} from '@tryghost/shade';
-import {useRouting} from '@tryghost/admin-x-framework/routing';
+import {useLocation, useNavigate} from '@tryghost/admin-x-framework';
 
-interface SidebarProps {
-    route: string;
-}
-
-const Sidebar: React.FC<SidebarProps> = ({route}) => {
-    const {updateRoute} = useRouting();
+const Sidebar: React.FC = () => {
+    const location = useLocation();
+    const navigate = useNavigate();
 
     return (
         <div className='sticky top-[102px] flex min-h-[calc(100vh-102px-32px)] w-[294px] flex-col border-l border-gray-200 dark:border-gray-950'>
             <div className='flex grow flex-col justify-between'>
                 <div className='flex w-full flex-col items-start gap-8 pl-4 pt-4'>
                     <div className='flex w-full flex-col gap-px'>
-                        <SidebarButton active={route === 'inbox'} onClick={() => updateRoute('inbox')}>
+                        <SidebarButton active={location.pathname === '/inbox' || location.pathname === '/'} onClick={() => navigate('/inbox')}>
```

Comment:
Would love to see this combined somehow - or wrapped into SidebarButton so it's just like 
`<SidebarButton route-"/inbox">` that would be cool

Reply:
Nice. I changed it as you recommended